### ISSUE TITLE
Imprecise description of server.tomcat.max-http-post-size

### ISSUE THREAD

**Issue by fg8dgrdh** [spring-projects/spring-framework#23763]
The document https://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html describes the following:

server.tomcat.max-http-post-size=2MB # Maximum size of the HTTP post content.

However this setting only affects post requests with FORM content, not all post requests. This can lead to confusion for myself and others (as per stackoverflow questions). A more precise description would be helpful, such as:

server.tomcat.max-http-post-size=2MB # Maximum size of the FORM content in an HTTP post request. Non-FORM post requests are unaffected by this property.

**Comment by sbrannen** [comment]
@fg8dgrdh, _this_ is the issue tracker for the core Spring Framework.

For Spring Boot, please use the following issue tracker: https://github.com/spring-projects/spring-boot/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc

Or perhaps @snicoll or @philwebb can transfer this issue.

**Comment by snicoll** [comment]
> Or perhaps @snicoll or @philwebb can transfer this issue.

Yes we can but please give us a bit of time to act on it before closing.
